spec/models/conversation_spec.rb

```diff
@@ -644,6 +648,26 @@
           expect(conversation.can_reply?).to be false
         end
       end
+
```

Comment:
let move all this to  can reply service spec .. and just have one method here ensure can_reply? calls our new service